plugin/inlay_hint.py

```diff
@@ -9,6 +10,36 @@
 import uuid
 
 
+class LspToggleInlayHintsCommand(LspWindowCommand):
+    capability = 'inlayHintProvider'
+
+    def run(self) -> None:
+        InlayHints.toggle(self.window)
+        status = "on" if InlayHints.are_enabled(self.window) else "off"
+        sublime.status_message("Inlay Hints are {}".format(status))
+        self.run_inlay_hints_for_current_window()
+
+    def run_inlay_hints_for_current_window(self) -> None:
```

Comment:
this extra method seems unnecessary. Adds extra 3 lines and not much else in clarity